### PR TITLE
Full Atomic Registry ToC and Content set

### PR DESCRIPTION
**Pull request by aweiteka**
A first pass at the proposed content set for Atomic Registry.

@adellape et al, it's pretty massive; comments welcome.

### CODE REVIEW

File: admin_guide/managing_projects.adoc [hunk 1]
```diff
@@ -221,6 +225,9 @@ be scheduled, because labels can only be set to one value.
 
 [[limit-projects-per-user]]
```
Comment (adellape) [inline admin_guide/managing_projects.adoc]: This ID has been separated from its heading below.

File: registry_quickstart/administrators/cli.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+:toc: macro
+:toc-title:
+:prewrap!:
+
+== Working with the Command Line Interface (CLI)
+
+{product-title} may be administered using a CLI client. The client is not
+installed on the host system but it is available by entering the *origin*
+container on the {product-title} host system. The client may be installed on a
+remote system. Refer to link:../cli_reference/index.html[documentation] for
```
Comment (adellape) [inline registry_quickstart/administrators/cli.adoc]: `link:../../`

File: registry_quickstart/administrators/cli.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
++
+----
+# oc whoami
+system:admin
+----
+
+The *system:admin* user is a special user that is only used for local
+administration tasks.
+
+See link:../cli_reference/index.html[CLI reference] for more details.
```
Comment (adellape) [inline registry_quickstart/administrators/cli.adoc]: `link:../../`

File: registry_quickstart/administrators/index.adoc [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+The {product-title} quickstart installation has been provided to get a system
+running as quickly as possible. Several non-production choices have been made to
+enable this installation experience.
+
+* **Single host**. Multi-host configuration is not supported with this deployment
+method.
+* **Non-HA**. It does not deploy a highly available configuration.
+* *Install Container*: {product-title} is container-based software. The
+quickstart installation is delivered and run using privileged Docker image. This
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: s/using/using a/

File: registry_quickstart/administrators/index.adoc [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+container deploys other containers. This custom installation method may not work
+on all systems.
+* *Container Management*. Effort has been made to standardize the operating environment so
+that production and non-production deployments are managed the same way.
+{product-title} uses link:https://github.com/openshift/origin/[OpenShift Origin-based]
+technology to deploy its services. This may be unexpected.
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: Can what "this" is referring to be clarified?

File: registry_quickstart/administrators/index.adoc [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+See link:../../architecture/index.html[architecture section] for more details.
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: s/See/See the/

?

File: registry_quickstart/administrators/index.adoc [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+* *Node Ports*: The installation uses a
+link:http://kubernetes.io/docs/user-guide/services/#type-nodeport[Kubernetes service feature]
+that exposes ports directly on the host. This eliminates dependencies on DNS
+configuration but is not a recommended method of exposing a Kubernetes service.
+To achieve this the default node port range has been expanded to include ports
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: this,

File: registry_quickstart/administrators/index.adoc [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+through 443. See value of *servicesNodePortRange* in configuration file
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: the value

File: registry_quickstart/administrators/index.adoc [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+*/etc/origin/master/master-config.yaml*.
+
+== Prerequisites
+
+* a single Linux system with Docker service
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: s/a/A/

File: registry_quickstart/administrators/index.adoc [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+* Allow traffic to TCP ports 443, 8443 and 5000.
+* Storage for images. By default registry images will use local storage at
+*/var/lib/origin/volumes*. It is recommended that external storage be provided
+at this mount point or configure the registry for cloud storage.
+
+[NOTE]
+====
+While {product-title} may run on a non-Red Hat-based distributions, most testing
+has been performed on Red Hat Enterprise Linux, Centos and Fedora, including Atomic host.
```
Comment (adellape) [inline registry_quickstart/administrators/index.adoc]: s/host/Host/

?

File: registry_quickstart/administrators/system_configuration.adoc [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+:toc: macro
+:toc-title:
+:prewrap!:
+
+== Configuring Registry Storage
+
+{product-title} may be administered using a CLI client. The client is not
+installed on the host system but it is available by entering the *origin*
+container on the {product-title} host system. The client may be installed on a
+remote system. Refer to link:../cli_reference/index.html[documentation] for
```
Comment (adellape) [inline registry_quickstart/administrators/system_configuration.adoc]: `link:../../`

File: registry_quickstart/administrators/system_configuration.adoc [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
++
+----
+# oc whoami
+system:admin
+----
+
+The *system:admin* user is a special user that is only used for local
+administration tasks.
+
+See link:../cli_reference/index.html[CLI reference] for more details.
```
Comment (adellape) [inline registry_quickstart/administrators/system_configuration.adoc]: `link:../../`

File: welcome/index.adoc [hunk 13]
```diff
@@ -32,7 +32,7 @@ a|link:../admin_guide/index.html[*Administrator Guide*]
 
 These topics describe how to use OpenShift as an administrator.
 
-a|link:../dev_guide/index.html[*Developer Guide*]
+a|link:../registry_quickstart/developers.html[*Developer Guide*]
```
Comment (adellape) [inline welcome/index.adoc]: Was this intentional? This is updating the welcome page for Origin to point to the registry developer quickstart. But it looks like you're taking care of the "welcome" material for AR in `index-atomic-registry.html`.

**Comment by adellape** [comment]
@aweiteka Some comments inline. At this point, I was mostly looking for any general breaking issues (markup, xrefs, etc), then for changes to shared content that might affect other builds. But I also included some minor comments re: typos or style as I went along, which can be taken/left.

File: registry_quickstart/administrators/index.adoc (hunk 5 above)
Comment (aweiteka) [inline registry_quickstart/administrators/index.adoc]: > Can what "this" is referring to be clarified?

This is presumptive. Removing.

File: welcome/index.adoc (hunk 13 above)
Comment (aweiteka) [inline welcome/index.adoc]: > Was this intentional?

No. Good catch! Reverted.

I discarded the Welcome topic in favor of the distro package index page.

**Comment by aweiteka** [comment]
@adellape I removed WIP title. I would be fine to merge this. There will be some follow-on PRs but smaller in scope. Let me know if there are more changes required.

**Comment by adellape** [comment]
@aweiteka Changes LGTM. Can you squash to one commit? Then I'll merge. :bow:

**Comment by adellape** [comment]
Added a couple labels for some downstream OpenShift branches so they're picked in next week's publish (just to keep the shared topics in sync). No revhistory needed.

**Comment by adellape** [comment]
Squashed. :shipit:

**Pull request merged**